pylint_odoo/checkers/no_modules.py

```diff
@@ -261,6 +274,15 @@ def visit_dict(self, node):
                 self.add_message('license-allowed',
                                  node=node, args=(license,))
 
+            # Check version format
+            version_format = manifest_dict.get('version', None)
```

Comment:
Change the code:
```dif
-            version_format = manifest_dict.get('version', None)
+           version_format = manifest_dict.get('version', '')
```
To avoid follow error:
<img width="749" alt="screen shot 2016-04-06 at 9 08 52 pm" src="https://cloud.githubusercontent.com/assets/6644187/14338344/e078cf6c-fc3b-11e5-811a-891216de724b.png">